src/main/scala/io/qbeast/spark/internal/sources/catalog/QbeastCatalogUtils.scala

```diff
@@ -151,6 +152,56 @@ object QbeastCatalogUtils {
     }
   }
 
+  /**
+   * Creates the Delta Log with Qbeast Metadata
+   *
+   * TODO: Right now is made in two steps:
+   *   1. Creates an empty dataframe and save it in @tableLocation 2. Converts the Delta Table to
+   *      Qbeast Table
+   *
+   * It is executed like that because we do not have access to methods in the
+   * CreateDeltaTableCommand, neither we can delegate the creation to that object Otherwise, the
```

Comment:
What's the reason we can't delegate the table creation to `CreateDeltaTableCommand?` We can simplify the process quite a bit.

Reply:
Because it will create a table in the catalog as "delta".

Reply:
And when the same method tries to create the Qbeast Table, it will fail because it already exists.

Reply:
Here: https://github.com/delta-io/delta/blob/455dbacaf456158644bcbe8b0b13bd4eae65c974/spark/src/main/scala/org/apache/spark/sql/delta/commands/CreateDeltaTableCommand.scala#L624-L650

Reply:
I can try to do the command with a table by path.... I will check!

Reply:
Still the same error... Nevertheless, we will dedicate a separate issue for that #371